Update changelog for v7.0.0b2

sirosen opened the pull request:
I was thinking that I would like to release v7.0.0b2 soon and I noticed that I haven't been putting in changelog entries the way I should.

This just gets the changelog up to date, and I'm happy to release 7.0.0b2 whenever this merges if that sounds good.

sirosen committed:
Update changelog for v7.0.0b2

lafrech reviewed (commented):
Sorry, I totally forgot about this.

Good.

Just a nitpick:

This part sounds redundant as it is repeated twice. 

> FalconParser` now defaults to the new `media` location

I think I understand the logic:

- feature: new media parser
- change: media is default

If so, we could keep it in two places as

- feature: new media parser (don't mention default)
- change: media is default

or merge it into:

- feature: new media parser, which becomes default.

I don't mind, both are totally valid, no need to overthink.

Feel free to merge after or even without rewording.

sirosen commented:
No worries! I figured you might be off for a few weeks.

I'll remove mention of "default" from the "feature" section and merge. I wanted to highlight it as a change since it could be backwards-incompatible for some people (at least in theory).

sirosen committed:
Minor tweak to 7.0.0b2 changelog notes

Clarify the distinction between the two notes on FalconParser:
- define `media`
- use `media` by default
